modules/develop/pages/data-transforms/deploy.adoc

```diff
@@ -65,19 +63,21 @@ rpk transform deploy --file=https://my-site/my-transform.wasm
 
 To make changes to an existing transform function:
 
-. xref:develop:data-transforms/build.adoc[Make you changes to the code].
+. xref:develop:data-transforms/build.adoc[Make your changes to the code].
 . <<build, Rebuild>> the Wasm binary.
 . <<deploy, Redeploy>> the Wasm binary to the same Redpanda cluster.
++
+If you provided configuration flags on the command line, those can be overwritten when you redeploy the Wasm binary.
```

Comment:
nitpick: more strict

```suggestion
If needed, deploy-time configuration overrides must be provided every time you deploy a Wasm binary. Otherwise they will be overwritten by default values and/or the contents of `transform.yaml`.
```

or some such. that's sort of clumsy.